venom.go

```diff
@@ -231,49 +236,24 @@ func (v *Venom) registerUserExecutors(ctx context.Context, name string, vars map
 			}
 			v.executorFileCache[f] = btes
 		}
-
-		varsFromInput, err := getUserExecutorInputYML(ctx, btes)
-		if err != nil {
-			return err
-		}
-
-		// varsFromInput contains the default vars from the executor
-		var varsFromInputMap map[string]string
-		if len(varsFromInput) > 0 {
-			varsFromInputMap, err = DumpStringPreserveCase(varsFromInput)
-			if err != nil {
-				return errors.Wrapf(err, "unable to parse variables")
-			}
+		executorName, _ := getExecutorName(btes)
+		if len(executorName) == 0 {
+			fileName := filepath.Base(f)
+			executorName = strings.TrimSuffix(fileName, filepath.Ext(fileName))
```

Comment:
The name of user executor is the attribute in the file. It should not be defined by the filename.

In the read, we can see that the user executor is defined in the file lib/customA.yml, but the executor name is executor.